csrc/backend_ops/onnxruntime/nms_rotated/nms_rotated.cpp

```diff
@@ -0,0 +1,384 @@
+// Copyright (c) OpenMMLab. All rights reserved
+#include "nms_rotated.h"
+
+#include <assert.h>
+
+#include <algorithm>
+#include <cmath>
+#include <iostream>
+#include <iterator>
+#include <numeric>  // std::iota
+#include <vector>
+#include <cassert>
+
+#include "ort_utils.h"
+
+namespace mmdeploy {
+
+struct RotatedBox {
+  float x_ctr, y_ctr, w, h, a;
+};
+struct Point {
```

Comment:
Names like `Point`, `dot_2d`, `cross_2d` are very common and may lead to name collsion in the future. Put op specific internal implementations inside a anonymous namespace to force internal linkage for them, like
```C++
namespace {

struct RotatedBox {...};
struct Point {...};

// ...

}

// and use them normally in the same cpp file
```